src/SourceBuild/content/eng/Version.Details.xml

```diff
@@ -3,9 +3,9 @@
   <ProductDependencies>
   </ProductDependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="9.0.0-beta.24075.2">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="9.0.0-beta.24081.4">
       <Uri>https://github.com/dotnet/arcade</Uri>
-      <Sha>1d8f27f89c3b167f63e28e73a3d9ab345e81d310</Sha>
+      <Sha>438a8e2488313fb3aa1b24a741a85c2669ee7e0d</Sha>
```

Comment:
For https://github.com/dotnet/installer/pull/18498, we need at least https://github.com/dotnet/arcade/commit/d82d6eb7c6aa6c737a83cbf3141d19db02a00006 but this SHA points to the older commit https://github.com/dotnet/arcade/commit/438a8e2488313fb3aa1b24a741a85c2669ee7e0d

Reply:
You can only bootstrap with what's available in installer. https://github.com/dotnet/arcade/commit/d82d6eb7c6aa6c737a83cbf3141d19db02a00006 hasn't reach installer yet so it's not possible to include that change.

Reply:
Interesting. We need a re-bootstrap to get https://github.com/dotnet/installer/pull/18498 in which brings that commit in. Sounds like we are hitting a dependency cycle here.

Reply:
Submitted https://github.com/dotnet/arcade/pull/14452 to avoid the cycle.